lib/shoryuken/manager.rb

```diff
@@ -173,11 +173,10 @@ def next_queue
       # get/remove the first queue in the list
       queue = @queues.shift
 
-
-      unless Shoryuken.workers.include? queue
+      unless defined?(::ActiveJob) || Shoryuken.workers.include?(queue)
```

Comment:
I had this in there originally because I found that only jobs on the default queue were being processed. However, I just tried getting rid of this now and all queues are processing, so it must have been unrelated.

Good catch! Will update.